src/rgw/rgw_opa.cc

```diff
@@ -45,6 +45,7 @@ int rgw_opa_authorize(RGWOp *& op,
   jf.dump_string("params", s->info.request_params.c_str());
   jf.dump_string("request_uri_aws4", s->info.request_uri_aws4.c_str());
   jf.dump_string("object_name", s->object.name.c_str());
+  jf.dump_string("user_access_key", s->auth.s3_postobj_creds.access_key);
```

Comment:
this doesn't compile because access_key is a private member of s3_postobj_creds

Reply:
I think it gets fixed in the new commit